### DISCUSSION TITLE
Edge TX Buddy Bricked my TX16S

### DISCUSSION THREAD

**Discussion by HeavyJohnny** [EdgeTX/edgetx#4877] [Q&A]
Help please,  Edge TX Buddy froze at 0% erasing current firmware. Now my TX is bricked and I don't know what to do.  The current version was 2.7.1. I followed the steps provided (https://manual.edgetx.org/installing-and-updating-edgetx/update-from-opentx-to-edgetx-1) in order to update to the latest version. I am unable to power on the TX.

**Comment by JrBoy66** [comment]
Have you tried taking your SD card out of your radio(powered off obviously) and seeing if you can install the firmware and sd contents directly on the card via a your SD slot on the computer?

**Reply by pfeerick** [reply]
Unfortunately that won't help if the erase really did start, and clobbered the bootloader. However, if the bootloader is still intact (i.e. hold horizontal trims inwards, power on radio, get bootloader menu), copying the files to SD and updating that way should be fine. i.e. using the other install guide at https://manual.edgetx.org/installing-and-updating-edgetx/migrate-from-opentx-to-edgetx-using-the-bootloader

If the erase really did start / bootloader won't start, this is the next place to go: https://github.com/EdgeTX/edgetx/wiki/Unbrick-your-radio
